### DISCUSSION TITLE
DIANN error with Methyl modification (unimod:34)

### DISCUSSION THREAD

**Discussion by aleksandrabinek** [vdemichev/DiaNN#646] [Q&A]
I’ve used DIA-NN multiple times to do library searches against libraries with modifications, this time round however it seems to have a problem with methyl mod (unimod:34), and it cannot finish the run.
Please see the log attached, I can provide the library file too if needed. Would you be able to help figuring out why this error occurred?
Thank you
![image](https://user-images.githubusercontent.com/129230234/228379360-0452cb1f-ce3f-48ef-b95f-4b418efda243.png)
[03_14_2023 17_08_51_log.txt](https://github.com/vdemichev/DiaNN/files/11094615/03_14_2023.17_08_51_log.txt)

**Comment by vdemichev** [comment]
Setting library generation strategy to "IDs, RT and IM profiling" or declaring the modification with --mod will address this.
